recipes/mac-and-cheese.qmd

```diff
@@ -0,0 +1,94 @@
+---
+title: "Mac & Cheese"
+author: Colleen Dollst
+theme: solar
+format: html
+editor: visual
+---
```

Comment:
@cdollst, could you please remove the lines specifying the `theme`, `format` and `editor` from the YAML header? Thank you!